tests/OSPSuite.HelpersForTests/ModuleHelperForSpecs.cs

```diff
@@ -0,0 +1,173 @@
+﻿using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+using static OSPSuite.Core.Domain.Constants;
+using static OSPSuite.Helpers.ConstantsForSpecs;
+
+namespace OSPSuite.Helpers
+{
+   public class ModuleHelperForSpecs
+   {
+      private readonly IObjectBaseFactory _objectBaseFactory;
+      private readonly ISolverSettingsFactory _solverSettingsFactory;
+      private readonly IOutputSchemaFactory _outputSchemaFactory;
+      private readonly ISpatialStructureFactory _spatialStructureFactory;
+      private readonly IParameterFactory _parameterFactory;
+      private readonly INeighborhoodBuilderFactory _neighborhoodFactory;
+
+      public ModuleHelperForSpecs(IObjectBaseFactory objectBaseFactory, ISolverSettingsFactory solverSettingsFactory, IOutputSchemaFactory outputSchemaFactory, ISpatialStructureFactory spatialStructureFactory, IParameterFactory parameterFactory, INeighborhoodBuilderFactory neighborhoodFactory)
+      {
+         _objectBaseFactory = objectBaseFactory;
+         _solverSettingsFactory = solverSettingsFactory;
+         _outputSchemaFactory = outputSchemaFactory;
+         _spatialStructureFactory = spatialStructureFactory;
+         _parameterFactory = parameterFactory;
+         _neighborhoodFactory = neighborhoodFactory;
+      }
+
+      public SimulationConfiguration CreateSimulationConfiguration()
+      {
+         var simulationConfiguration = new SimulationConfigurationForSpecs
+         {
+            SimulationSettings = createSimulationConfiguration(),
+         };
+
+         var module1 = createModule1();
```

Comment:
We will create multiple modules etc. I copied the first structure from the other integration tests but we do not have to